extensions/positron-assistant/src/participants.ts

```diff
@@ -424,6 +424,10 @@ abstract class PositronAssistantParticipant implements IPositronAssistantPartici
 /** The participant used in the chat pane in Ask mode. */
 class PositronAssistantChatParticipant extends PositronAssistantParticipant implements IPositronAssistantParticipant {
 	id = ParticipantID.Chat;
+
+	protected override async getSystemPrompt(request: vscode.ChatRequest): Promise<string | undefined> {
```

Comment:
I wonder if we also want this in the `EditParticipant` (used in the chat pane in edit mode)?